governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md

```diff
@@ -0,0 +1,69 @@
+---
+title: PPG - Open Market Committee Proposal - January 17, 2022
+summary: Minor stability fee decreases on ETH-A and wSTETH-A, a handful of debt ceiling decreases, expanding the Aave D3M and the UniswapV3-Gelato DAI-USDC 0.01% fee vault.
+discussion_link: https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-2022-01-07/12522
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Rates
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-01-17T16:00:00
+end_date: 2022-01-20T16:00:00
+---
+# Poll: PPG - Open Market Committee Proposal - January 17, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) will be active for three days beginning on Monday, January 17 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+MKR Token Holders may vote in this poll to express support or opposition to the following parameter changes: 
+
+**Stability Fee Decreases**
+* Decrease the ETH-A [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 2.75% to 2.5%.
+* Decrease the WSTETH-A [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 4% to 3%.
+
+**Debt Ceiling Decreases**
+* Decrease the LINK-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 140 million DAI to 100 million DAI.
+* Decrease the YFI-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 130 million DAI to 50 million DAI.
+* Decrease the UNI-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 50 million DAI to 25 million DAI.
+* Decrease the UNIV2UNIETH-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 20 million DAI to 5 million DAI.
+* Decrease the GUSD-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) from 5 million DAI to zero.
+
+**DIRECT-AAVEV2-DAI (Aave D3M) Changes**
+* Decrease the DIRECT-AAVEV2-DAI Target Borrow Rate from 3.9% to 3.75%.
```

Comment:
maybe link to [bar](https://github.com/makerdao/dss-direct-deposit#configuration)

Reply:
Yeah, better than nothing.